ui/components/app/permissions-connect-header/permissions-connect-header.component.js

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import SiteOrigin from '../../ui/site-origin/site-origin';
+import SiteOrigin from '../../ui/site-origin';
+import SnapsAuthorshipPill from '../flask/snaps-authorship-pill';
```

Comment:
This import will fail on non-Flask builds because modules under `flask` directories are excluded. So this will need to be behind a code fence